jwst/resample/resample.py

```diff
@@ -173,6 +214,24 @@ def resample_many_to_many(self):
             log.info(f"{len(exposure)} exposures to drizzle together")
             for img in exposure:
                 img = datamodels.open(img)
+
+                input_pixflux_area = img.meta.photometry.pixelarea_steradians
+                if input_pixflux_area:
+                    img.meta.wcs.array_shape = img.data.shape
+                    input_pixel_area = _compute_image_pixel_area(img.meta.wcs)
+                    if input_pixel_area is None:
+                        raise ValueError(
+                            "Unable to compute input pixel area from WCS of input "
+                            f"image {repr(img.meta.filename)}."
+                        )
+                    if self.input_pixscale0 is None:
+                        self.input_pixscale0 = np.rad2deg(np.sqrt(input_pixel_area))
+                        if self._recalc_pscale_ratio:
+                            self.pscale_ratio = self.pscale / self.input_pixscale0
+                    iscale = np.sqrt(input_pixflux_area / input_pixel_area)
```

Comment:
Just curious, for my own knowledge, why the output image flux level has to be rescaled manually by the ratio of pixel areas. Why doesn't the drizzling process take care of this?

Reply:
I think these are some very very good questions!

1. >  Why doesn't the drizzling process take care of this?